tools/cpackget/docs/cpackget.md

```diff
@@ -19,101 +27,211 @@ Usage:
+  -R, --pack-root path     Specifies path of the central CMSIS-Pack root directory, default: CMSIS_PACK_ROOT environment variable
   -q, --quiet              Run cpackget silently, printing only error messages
   -v, --verbose            Sets verboseness level: None (Errors + Info + Warnings), -v (all + Debugging). Specify "-q" for no messages
   -V, --version            Prints the version number of cpackget and exit
 
 Use "cpackget [command] --help" for more information about a command.
 ```
 
-For example, if one wanted help removing a pack, running `cpackget pack rm --help` would print out useful information on the subject.
+Usage help for a specific command available, with for example: `cpackget rm --help`.
+
+## Specify central CMSIS-PACK directory
+
+`cpackget` is compatible with other CMSIS-Pack management tools, such as the Pack Installer available in MDK or Eclipse variants. There are two ways to specify the central CMSIS-PACK root directory:
+
+1. With the `CMSIS_PACK_ROOT` environment variable. Refer to [Installation - Environment Variables](Installation.md#environment-variables).
+
+2. with the option `--pack-root <path>`, for example:
 
-### Specify the working pack root folder
+   ```yml
+   $ cpackget add Vendor.PackName -path-root /MyLocal/Packs
+   ```
 
-If cpackget is going to work on an existing pack root folder, there are two ways to specify it:
+>NOTE: As the various tools of the CMSIS-Toolbox rely all on this central CMSIS-Pack directory, it is recommended to use the `CMSIS_PACK_ROOT` environment variable. 
 
-1. `export CMSIS_PACK_ROOT=path/to/pack-root; cpackget pack add ARM.CMSIS`
-2. `cpackget --pack-root path/to/pack-root pack add ARM.CMSIS`
+## Initialize central CMSIS-Pack directory
 
-To create a new pack root folder with an up-to-date index file of publicly available Open-CMSIS-Pack packs run:
+CMSIS-Packs are typically distributed via a public web service, that offers a [**Pack Index File**](https://open-cmsis-pack.github.io/Open-CMSIS-Pack-Spec/main/html/packIndexFile.html) of available software packs. To initialize the central CMSIS-Pack directory run the command:
 
+```yml
+$ cpackget init https://www.keil.com/pack/index.pidx
 ```
-$ cpackget init --pack-root path/to/new/pack-root https://www.keil.com/pack/index.pidx
+
+This command creates in the central CMSIS-PACK directory the following sub-directories.
+
+Sub-Directory   | Content
+:---------------|:------------------------
+`.Web`          | [**Pack Index File**](https://open-cmsis-pack.github.io/Open-CMSIS-Pack-Spec/main/html/packIndexFile.html) of a public web service and `*.PDSC` files.
+`.Download`     | Packs that are installed from a web service. Stores `*.PDSC` pack description file, `*.pack` content file, and related license information.
+`.Local`        | Index file `local_repository.pidx` that points to local installations for development of a software pack.
```

Comment:
Any pack that I install via a pack-file and that is not listed in the index.pidx, will have it's pdsc file copied to .Local.
For implementing a "check-for-updates" the url vendor and packname will be used to download the pdsc file, which may contain a new version which could be downloaded and installed e.g. from a user's/company's drive.